include/deepstream/lib/json-handler.hpp

```diff
@@ -0,0 +1,134 @@
+#ifndef DEEPSTREAM_LIB_JSON_HANDLER_HPP
```

Comment:
i would call this something like deepstream_type_serializer since it does more than just json parsing.